esp/src/src-react/routes.tsx

```diff
@@ -83,10 +83,8 @@ export const routes: RoutesEx = [
             { path: "", action: (ctx) => import("./components/Workunits").then(_ => <_.Workunits filter={parseSearch(ctx.search) as any} />) },
             { path: "/dashboard", action: (ctx) => import("./components/WorkunitsDashboard").then(_ => <_.WorkunitsDashboard filterProps={parseSearch(ctx.search) as any} />) },
             { path: "/:Wuid", action: (ctx, params) => import("./components/WorkunitDetails").then(_ => <_.WorkunitDetails wuid={params.Wuid as string} />) },
-            { path: "/:Wuid/outputs/:Name", action: (ctx, params) => import("./components/Result").then(_ => <_.Result wuid={params.Wuid as string} resultName={params.Name as string} filter={parseSearch(ctx.search) as any} />) },
-            { path: "/:Wuid/metrics", action: (ctx, params) => import("./components/Metrics").then(_ => <_.Metrics wuid={params.Wuid as string} />) },
-            { path: "/:Wuid/metrics/:id", action: (ctx, params) => import("./components/Metrics").then(_ => <_.Metrics wuid={params.Wuid as string} selection={(params.id as string)} />) },
             { path: "/:Wuid/:Tab", action: (ctx, params) => import("./components/WorkunitDetails").then(_ => <_.WorkunitDetails wuid={params.Wuid as string} tab={params.Tab as string} />) },
+            { path: "/:Wuid/:Tab/:State", action: (ctx, params) => import("./components/WorkunitDetails").then(_ => <_.WorkunitDetails wuid={params.Wuid as string} tab={params.Tab as string} state={(params.State as string)} />) },
```

Comment:
This one should precede the more generic /:Wuid/:Tab route

Reply:
FWIW the additional "/" is enough to differentiate them, but for consistency I agree.

Reply:
@jeclrsg just checked the rest of the file and this code is consistent with the rest, so I am inclined to leave as is for now.  Can you re-review?